Resources/Localization/es-ES.lproj/Localizable.strings

```diff
@@ -3233,3 +3233,6 @@
 
 /* New Connection Error informative message */
 "Failed to create new database connection window. Please restart Sequel Ace and try again." = "Failed to create new database connection window. Please restart Sequel Ace and try again.";
+
+/* Warning message during connection in case the variable skip-show-database is set to ON */
```

Comment:
For future - you need to add it just to english, process will take care of the rest of the languages, but thanks! :)